.vscode/settings.json

```diff
@@ -1,3 +1,5 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
```

Comment:
This we should perhaps discuss or align - we've discussed weather or not to share vscode settings but forget what we agreed on.

The typescript setting I think would be great to share tho!

Reply:
I understand. I'm not aware of a way to set it up as a recommendation. So I think it would be better to revert this and keep these settings about the editor as a personal preference :)